src/testing/builder/CategoriesMapModelBuilder.js

```diff
@@ -95,18 +96,17 @@ class CategoriesMapModelBuilder {
     const categories = []
     const resourceCache = {}
 
-    for (let i = 0; i < this._arity; i++) {
-      this._addChildren(new CategoryModel({
-        path: '/augsburg/de',
-        title: 'augsburg',
-        content: '',
-        order: -1,
-        availableLanguages: new Map(),
-        thumbnail: 'no_thumbnail',
-        parentPath: '',
-        lastUpdate: moment('2017-11-18T19:30:00.000Z', moment.ISO_8601)
-      }), 0, categories, resourceCache, 0, [])
-    }
+    this._categories = []
+    this._addChildren(new CategoryModel({
+      path: '/augsburg/de',
+      title: 'augsburg',
+      content: '',
+      order: -1,
+      availableLanguages: new Map(),
+      thumbnail: 'no_thumbnail',
+      parentPath: '',
+      lastUpdate: moment('2017-11-18T19:30:00.000Z', moment.ISO_8601)
+    }), resourceCache, 0)
 
     return { categories: new CategoriesMapModel(categories), resourceCache }
```

Comment:
As far as I can see, `categories` is always an empty array. You probably mean to use `this._categories` and remove the `categories` variable?

Reply:
This definitely needs tests :D